backend/account/tests/test_views.py

```diff
@@ -173,3 +174,97 @@ def test_invalid_oauth_properties(self, mock_verify_oauth2_token):
             **self.header,
         )
         self.assertEqual(response.status_code, 401)
+
+
+class FriendTests(TestCase):
```

Comment:
These tests should mock the business layer functions they rely on (e.g. `test_add_friend` should mock the business `add_friend` function return value)